pkg/render/node.go

```diff
@@ -1368,6 +1842,10 @@ func (c *nodeComponent) nodeMetricsService() *v1.Service {
 	}
 }
 
+func (c *nodeComponent) useVpp() bool {
+	return c.cr.CalicoNetwork != nil && c.cr.CalicoNetwork.VPP != nil
```

Comment:
The check here should be similar to the one for bpf:
```
return c.cr.CalicoNetwork != nil &&
		c.cr.CalicoNetwork.LinuxDataplane != nil &&
		*c.cr.CalicoNetwork.LinuxDataplane == operatorv1.LinuxDataplaneVPP 
```
with  LinuxDataplaneVPP = "vpp"